docs/faq.md

```diff
@@ -189,7 +189,7 @@ fixed CRD if you want this feature to work at all.
 The controller of the `SealedSecret` resource may expose the status condition on resource it provisioned. Since
 version `v2.0.0` ArgoCD picks up that status condition to derive a health status for the `SealedSecret`.
 
-Versions before `v0.15.0` of the `SealedSecret` controller are affected by an issue regarding this status
+Versions before `v0.16.0` of the `SealedSecret` controller are affected by an issue regarding this status
```

Comment:
Looks like default will be updated in v0.17.0, not v0.16.0 . Should we want to v0.17.0 release and then update documentation?

Reply:
If people using `v.0.16.0` read this dcumentation, they may be confused. In fact, I was confused.